src/main/lrsql/util/oidc.clj

```diff
@@ -0,0 +1,20 @@
+(ns lrsql.util.oidc
+  "OpenID Connect Utilities"
+  (:require [clojure.spec.alpha :as s]
+            [clojure.string :as cs]
+            [com.yetanalytics.pedestal-oidc.discovery :as disco]
+            [lrsql.spec.config :as config]))
```

Comment:
Looks like `clojure.string` and `lrsql.spec.config` are unused in this namespace.